vignettes/example_script.Rmd

```diff
@@ -207,5 +219,18 @@ plot_standard_curve_analyte_with_model(plate, model, log_scale = c("all"))
 plot_standard_curve_analyte_with_model(plate, model, log_scale = c("all"), over_max_extrapolation = 100000)
 ```
 
+### nMFI
+In some cases, the RAU values cannot be properly calculated. It may happen, for instance when the MFI values of tests samples are way higher than the standard curve samples. In that case, to avoid extrapolation, but to be still able to compare the samples across the plates, we introduced a new unit called nMFI (Normalized MFI). The nMFI is calculated as the MFI value of the test sample divided by the MFI value of the standard curve sample with selected dilution value.
```

Comment:
`
```suggestion
In some cases, the RAU values cannot be reliably calculated. It may happen, for instance when the MFI values of tests samples are way higher than the standard curve samples. In that case, to avoid extrapolation, but to be still able to compare the samples across the plates, we introduced a new unit called nMFI (Normalized MFI). The nMFI is calculated as the MFI value of the test sample divided by the MFI value of the standard curve sample with selected dilution value.
```